homeassistant/components/pushbullet/manifest.json

```diff
@@ -1,9 +1,11 @@
 {
   "domain": "pushbullet",
   "name": "Pushbullet",
+  "dependencies": [],
   "documentation": "https://www.home-assistant.io/integrations/pushbullet",
   "requirements": ["pushbullet.py==0.11.0"],
```

Comment:
The library has been archived and isn't maintained anymore. I'd recommend finding or building another library.

Reply:
I just noticed this as my PR was submitted before the library got archived. I will try to find another library or build one if necessary but can we keep it for this PR?

Reply:
Yes, it's ok for now.